components/eamxx/CMakeLists.txt

```diff
@@ -197,7 +205,12 @@ set(NetCDF_Fortran_PATH ${DEFAULT_NetCDF_Fortran_PATH} CACHE FILEPATH "Path to n
 set(NetCDF_C_PATH ${DEFAULT_NetCDF_C_PATH} CACHE FILEPATH "Path to netcdf C installation")
 set(SCREAM_MACHINE ${DEFAULT_SCREAM_MACHINE} CACHE STRING "The CIME/SCREAM name for the current machine")
 option(SCREAM_MPI_ON_DEVICE "Whether to use device pointers for MPI calls" ON)
+
 option(SCREAM_ENABLE_MAM "Whether to enable MAM aerosol support" ON)
+if (Kokkos_ENABLE_SYCL)
+  option(SCREAM_ENABLE_MAM "Whether to enable MAM aerosol support" OFF)
```

Comment:
I'm not sure this will work as you intend. You may have to simply do `set(SCREAM_ENABLE_MAM OFF)`

Reply:
thanks, but that one unlike SET worked.
```
//Whether to enable MAM aerosol support
SCREAM_ENABLE_MAM:BOOL=OFF
```
if you have a preference i will change it.

Reply:
I'm puzzled. This
```cmake
option(BLAH "" ON) 
message ("BLAH: ${BLAH}")
option(BLAH "" OFF)
message ("BLAH: ${BLAH}")
```
prints
```
BLAH: ON
BLAH: ON
```
Are you sure you did not have another line _before_ those that set SCREAM_ENABLE_MAM to OFF? Maybe in the mach file? Also, was it a clean build?

My preference is for using `set(... CACHE ..)` (or `option(...)`) only _once_.

Reply:
I found this in Professional CMake:
```
An important difference between normal and cache variables is that the set() command will only
overwrite a cache variable if the FORCE keyword is present, unlike normal variables where the set()
command will always overwrite a pre-existing value. The set() command acts more like set-if-not-
set when used to define cache variables, as does the option() command (which has no FORCE
capability). The main reason for this is that cache variables are primarily intended as a
customization point for developers.
```
which makes me think something else may have caused SCREAM_ENABLE_MAM=OFF in your cache file...